e2e/rbd.go

```diff
@@ -366,6 +366,59 @@ var _ = Describe("RBD", func() {
 					}
 				})
 			}
+			By("validate RBD migration+static Block PVC Deletion", func() {
+				// create monitors hash by fetching monitors from the cluster.
+				mons, err := getMons(rookNamespace, c)
+				if err != nil {
+					e2elog.Failf("failed to get monitors %v", err)
+				}
+				mon := strings.Join(mons, ",")
+				inClusterID := getMonsHash(mon)
+				customInfo := map[string]string{
+					"clusterID-1": "subvolgrp1",
+					// the monitors hash entry in the configmap.
+					inClusterID: "subvolgrp2",
+				}
+				// create custom configmap
+				err = createCustomConfigMap(f.ClientSet, rbdDirPath, customInfo)
+				if err != nil {
+					e2elog.Failf("failed to create configmap with error %v", err)
+				}
+				err = createRBDStorageClass(f.ClientSet, f, "migrationsc", nil, nil, deletePolicy)
+				if err != nil {
+					e2elog.Failf("failed to create storageclass with error %v", err)
+				}
+				// restart csi pods for the configmap to take effect.
+				err = deletePodWithLabel("app in (ceph-csi-rbd, csi-rbdplugin, csi-rbdplugin-provisioner)",
+					cephCSINamespace, false)
+				if err != nil {
+					e2elog.Failf("failed to delete pods with labels with error %v", err)
+				}
+				// wait for csi pods to come up
+				err = waitForDaemonSets(rbdDaemonsetName, cephCSINamespace, f.ClientSet, deployTimeout)
+				if err != nil {
+					e2elog.Failf("timeout waiting for daemonset pods with error %v", err)
+				}
+				err = waitForDeploymentComplete(rbdDeploymentName, cephCSINamespace, f.ClientSet, deployTimeout)
+				if err != nil {
+					e2elog.Failf("timeout waiting for deployment to be in running state with error %v", err)
+				}
+
+				err = validateRBDStaticMigrationPVDeletion(f, rawAppPath, true)
```

Comment:
pass created sc as input parameter to the function and use it to create PV/PVC object

Reply:
Done.